### ISSUE TITLE
release0.3.1 : "1:13404" Two ALT alleles , but AC_Het has 3 values.

### ISSUE THREAD

**Issue by lindenb** [konradjk/exac_browser#256]
via https://www.biostars.org/p/183713/

is there an error in ftp://ftp.broadinstitute.org/pub/ExAC_release/release0.3.1/ExAC.r0.3.1.sites.vep.vcf.gz ?

for 1:13404 there are two ALT alleles,  but AC_Het(Number='A') has three values.

```
##INFO=<ID=AC_Het,Number=A,Type=Integer,Description="Adjusted Heterozygous Counts">
(...)
1   13404   .   G   A,T 136.58  VQSRTrancheSNP99.60to99.80  (...)AC_Het=0,1,0;
```

**Comment by roland-ewald** [comment]
+1

**Comment by konradjk** [comment]
This is now fixed with release 1.0 - sorry for the delay!
